pkg/danmep/danm.go

```diff
@@ -0,0 +1,19 @@
+package danmep
+
+import (
+	danmtypes "github.com/nokia/danm/crd/apis/danm/v1"
+	clientset "github.com/nokia/danm/crd/client/clientset/versioned"
+	danm "github.com/nokia/danm/pkg/danmep"
+	"github.com/nokia/danm/pkg/ipam"
+)
+
+type danmDeleteServiceImpl struct {}
+
+func (h *danmDeleteServiceImpl) WasIPAllocatedByMe(danmClient clientset.Interface, ep *danmtypes.DanmEp, dnet *danmtypes.DanmNet) bool {
+	return ipam.WasIpAllocatedByDanm(ep.Spec.Iface.Address, dnet.Spec.Options.Cidr) ||
```

Comment:
hmm, hmm
because of the dual stack and the possibility to use different IPAMs the error handling ultiately needs to be moved to the framework level
i.e. each handler report errors, and DanmEp deletion only once, after all handlers have finished, only if no error was observed

Note: I'm okay with this not being done now, no worries :D would appreciated a TODO about it tho so we wont forget